### ISSUE TITLE
[dataclasses] I3SuperDSTTrigger pybindings (Trac #1965)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1957]
Complete the pybindings for I3SuperDSTTrigger, so we stop getting this message:

```(No Python class registered for C++ class I3SuperDSTTrigger)```

Migrated from https://code.icecube.wisc.edu/ticket/1965
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:03", 
    "description": "Complete the pybindings for I3SuperDSTTrigger, so we stop getting this message:\n\n{{{(No Python class registered for C++ class I3SuperDSTTrigger)}}}\n\n", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067243750755", 
    "component": "combo core", 
    "summary": "[dataclasses] I3SuperDSTTrigger pybindings", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-03-17T20:43:09", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170816T22:45:21: 
* david.schultz commented:

> In changeset 157649:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157649"
> add I3SuperDSTTrigger pybinidngs. closes #1957
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:03: 
* olivas changed milestone from "August 2017" to ""
* olivas commented:

> Milestone August 2017 deleted
